specification/common-types/resource-management/v2/privatelinks.json

```diff
@@ -107,6 +107,10 @@
           "$ref": "#/definitions/PrivateLinkResourceProperties",
           "description": "Resource properties.",
           "x-ms-client-flatten": true
+        },
+        "systemData": {
```

Comment:
please don't update common types, it's breaking change for all the RPs which refer to the v2/types.json

Reply:
> please don't update common types, it's breaking change for all the RPs which refer to the v2/privatelinks.json

Reverted it.